client/src/App.css

```diff
@@ -1,6 +1,7 @@
 * {
   font-family: "Nunito Sans", sans-serif;
   color: white;
+  scroll-behavior: smooth;
```

Comment:
This is a great solution and it looks fantastic in Google Chrome and Firefox, but unfortunately `scroll-behavior` is not supported at all in Safari. (If you scroll to the bottom of the [MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior) you can see the browser compatibility chart)

This is an issue that often comes up, where not all browsers support or respect JS or CSS the same way. To get around this, developers use something called a **polyfill**. A **polyfill** _fills_ in the gaps to create cross-browser functionality with wider support. 

This scrolling bug with Safari is very well documented, and this is a good workaround written by Dustan Kasten and Jeremias Menichelli. http://iamdustan.com/smoothscroll/

We can install that via NPM and follow the instructions to get it set up. It's also a good habit to credit other engineers that have written particularly helpful libraries or polyfills with a small comment in the JS that also contains the link to the github.